massa-node/src/main.rs

```diff
@@ -902,20 +902,25 @@ async fn stop(
             .expect("bootstrap server shutdown failed")
     }
 
-    // stop public API
-    api_public_handle.stop();
-
-    // stop private API
-    api_private_handle.stop();
-
-    // stop Massa API
-    api_handle.stop();
+    info!("Start stopping API's: gRPC, EXPERIMENTAL, PUBLIC, PRIVATE");
```

Comment:
You can also do this for stop all API in same time : 
```rust
    info!("Start stopping API's: gRPC, EXPERIMENTAL, PUBLIC, PRIVATE");

    // stop Massa gRPC API
    if let Some(handle) = grpc_handle {
        handle.stop();
    }

    join!(
        api_handle.stop(),
        api_public_handle.stop(),
        api_private_handle.stop(),
    )
    .await;

    info!("Stopped API's : gRPC, EXPERIMENTAL, PUBLIC, PRIVATE");

```

Reply:
Why not but we are trying to spot where we have an issue, which make sequential execution better

Reply:
in fact we should not have bug here :grimacing: . We can separate if bug appears. 

Or keep it like that the optimisation is not significant

Reply:
Even in the info message, there is an order